src/main/java/offeneBibel/parser/OffeneBibelParser.java

```diff
@@ -569,6 +614,7 @@ public Rule SecondaryContent() {
                      ParallelPassage(),
                      Note(),
                      Omission(),
+                     Sequence("“", createOrAppendTextNode("“")),
```

Comment:
Wenn ich mich recht erinnere, war das ein Zitat, das in einem SecondaryContent auf- und in einem späteren SecondaryContent eines späteren Verses zu ging. Bin mir aber nicht mehr sicher.

Denke du kannst die Stellen alle recht schnell finden, indem du sie einzeln auskommentierst und jeweils den Parser neu laufen lässt bis er hängt.

Falls er schon wieder nicht mehr durchläuft, hier ist eine Kopie des PageCache zum Zeitpunkt als er es noch tat: https://gist.github.com/schierlm/45bfbdc4b24c4fc24b7e#file-pagecache-tar-xz

Wenn was unklar bleibt, kann ich am Wochenende ja nochmal schauen, was das genau war.